api/client/service.proto

```diff
@@ -0,0 +1,50 @@
+syntax = "proto3";
+
+package nighthawk.client;
+
+import "api/client/options.proto";
+import "api/client/output.proto";
+
+message StartRequest {
+  CommandLineOptions options = 1;
+}
+
+/*
+TODO(oschaaf): Not implemented yet.
+*/
+message UpdateRequest {
+}
+
+/*
+TODO(oschaaf): Not implemented yet.
+*/
+message CancellationRequest {
+}
+
+message ExecutionRequest {
+  oneof command_specific_options {
+    StartRequest start_request = 1;
+    UpdateRequest update_request = 2;
+    CancellationRequest cancellation_request = 3;
+  }
+}
+
+/*
+Output is guaranteed to be set when success is true. When success is false,
+output may or may not be set, and error message is guaranteed to be set.
+*/
+message ExecutionResponse {
+  Output output = 1;
```

Comment:
This could be:
``
oneof status_specifier {
  Output output = 1;
  Error error = 2;
}
```

with the `error_message` a field in `Error`. I think there is something already defined like this in the WKT, see https://github.com/envoyproxy/envoy/blob/master/api/envoy/api/v2/discovery.proto#L190.
```

Reply:
Fixed